mapbox_vector_tile/encoder.py

```diff
@@ -229,6 +229,11 @@ def _get_feature_type(self, shape):
             return self.tile.LineString
         elif shape.type == 'Polygon' or shape.type == 'MultiPolygon':
             return self.tile.Polygon
+        elif shape.type == 'GeometryCollection':
+            raise ValueError('Encoding geometry collections not supported')
+        else:
+            raise ValueError('Cannot encode unknown geometry type: %s' %
+                             shape.type)
```

Comment:
I think so too, but I'd rather have dead code that would explicitly fail loudly this way, if only to make the code clearer. I thought about using an assertion instead, but opted for a `ValueError`. Do you think an assertion would have been more appropriate?